### PR TITLE
Two tests - both passing for server endpoints

### PR DESCRIPTION
**Pull request by elmarek**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -19,11 +19,13 @@ Promise.promisifyAll(require("mongoose"));
 
 // });
 
-app.get('/', function (req, res) {
+const status = app.get('/', function (req, res) {
```
Comment (Robinnnnnn) [inline server/index.js:22]: Great variable name here!

File: tests/getResponse.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+const request = require('supertest')
+const app = require('../server/index.js')
+
+jest.useFakeTimers();
+
+describe('Server Endpoints', () => {
```
Comment (Robinnnnnn) [inline tests/getResponse.test.js:7]: 2 tests in one!? Great!

File: server/index.js (hunk 1 above)
Comment (elmarek) [reply server/index.js:22]: Thought you might approve of this.

File: tests/getResponse.test.js (hunk 2 above)
Comment (elmarek) [reply tests/getResponse.test.js:7]: Im a coder, what can I say.

**Pull request merged**